libmamba/include/mamba/core/util_string.hpp

```diff
@@ -286,7 +398,7 @@ namespace mamba
         }
     }
 
-    // TODO(C++20) Take an input range and return a range
+    // TODO(C++20) This is a range
```

Comment:
Are we willing to change something here later with C++20? If yes, maybe specify what exactly as a comment, otherwise, remove the comment.